### PR TITLE
Add example of asp-area with a Razor Pages area

### PR DESCRIPTION
**Pull request by scottaddie**
Fixes https://github.com/aspnet/Docs/issues/9834

Also converted API ref links to the xref style.

[Internal Review Page](https://review.docs.microsoft.com/en-us/aspnet/core/mvc/views/tag-helpers/built-in/anchor-tag-helper?branch=pr-en-us-10048&view=aspnetcore-2.2#asp-area)

### CODE REVIEW

File: aspnetcore/mvc/views/tag-helpers/built-in/samples/TagHelpersBuiltIn/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -63,6 +63,19 @@
                 <!-- </snippet_AspArea> -->
             </td>
         </tr>
+        <tr>
+            <td>
+                <code>
+                    @Html.Raw(Html.Encode(@"<a asp-area=""Sessions"" asp-page=""/Index"">View Sessions</a>"))
```
[the remark this answers is not in the document]
Comment (scottaddie) [reply aspnetcore/mvc/views/tag-helpers/built-in/samples/TagHelpersBuiltIn/Views/Home/Index.cshtml]: Not sure I'm following. Do you have an example of what you mean?

[the remark this answers is not in the document]
Comment (guardrex) [reply aspnetcore/mvc/views/tag-helpers/built-in/samples/TagHelpersBuiltIn/Views/Home/Index.cshtml]:
```
&lt;a asp-area="Sessions" asp-page="/Index"&gt;View Sessions&lt;/a&gt;
```
🎤 drop :smile: ... wouldn't this do the same thing?

[the remark this answers is not in the document]
Comment (scottaddie) [reply aspnetcore/mvc/views/tag-helpers/built-in/samples/TagHelpersBuiltIn/Views/Home/Index.cshtml]: Yes, that would do the same thing. I think readability suffers with that approach.

[the remark this answers is not in the document]
Comment (guardrex) [reply aspnetcore/mvc/views/tag-helpers/built-in/samples/TagHelpersBuiltIn/Views/Home/Index.cshtml]: I'm such an 👴 *old geezer* 👴. In the olden days ... when the pyramids were built and code was written in hieroglyphics :smile: ... that's how we used to do it.

[the remark this answers is not in the document]
Comment (scottaddie) [reply aspnetcore/mvc/views/tag-helpers/built-in/samples/TagHelpersBuiltIn/Views/Home/Index.cshtml]: It's personal preference, really. I saw this as an opportunity to use framework features for encoding (what you'd do in real-world apps).

**Pull request merged**